tests/console/btrfsmaintenance.pm

```diff
@@ -65,16 +65,10 @@ sub run {
 
     die "no btrfs volume present" if (script_output('mount') !~ "btrfs");
 
-    # Check state of btrfsmaintenance-refresh units. Have to use (|| :) due to pipefail
-    validate_script_output('systemctl is-enabled btrfsmaintenance-refresh.path || true', qr/enabled/, fail_message => 'btrfsmaintenance-refresh.path is not enabled') unless is_sle("<15");
-    # Fixed in SP1:Update, but is out of general support
-    if (is_sle("<15-SP2")) {
-        validate_script_output('systemctl is-enabled btrfsmaintenance-refresh.service || true', qr/enabled/, fail_message => 'btrfsmaintenance-refresh.service is not enabled');
-        record_soft_failure('boo#1165780 - Preset is wrong and enables btrfsmaintenance-refresh.service instead of .path');
-    } else {
-        # Preset is correct, btrfsmaintenance-refresh.service dropped the [Install] section
-        validate_script_output('systemctl is-enabled btrfsmaintenance-refresh.service || true', qr/^static.*/, fail_message => 'btrfsmaintenance-refresh.service is not static');
-    }
+    # Check state of btrfsmaintenance-refresh units.
+    validate_script_output('systemctl is-enabled btrfsmaintenance-refresh.path || true', qr/enabled/, fail_message => "btrfsmaintenance-refresh.path must be enabled");
```

Comment:
You removed the `unless is_sle("<15")` part, I wonder whether this is still broken on SLE 12